lib/active_record/mysql/enum/mysql_adapter.rb

```diff
@@ -30,25 +30,33 @@ def native_database_types #:nodoc
         if Rails::VERSION::MAJOR < 5
           def type_to_sql(type, limit = nil, precision = nil, scale = nil, unsigned = nil, **_options) # :nodoc:
             if type.to_s == 'enum'
-              native = native_database_types[type]
-              column_type_sql = (native || {})[:name] || 'enum'
+              column_type =
+              if (native_database_type = native_database_types[type])
+                native_database_type[:name]
+              else
+                'enum'
+              end
```

Comment:
Lines 34-38 should be indented one stop from `column_type =`.

Reply:
And looking at the diff, I like their name `column_type_sql` better! Mind renaming `column_type` to `column_type_sql` here and below?